compiler/circle-tensordump/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 if(NOT TARGET mio_circle)
   return()
 endif(NOT TARGET mio_circle)
-
+set(HDF5_USE_STATIC_LIBRARIES OFF)
```

Comment:
is it possible to introduce a macro in compiler cmake and call that?

Reply:
It's possible but I think this setting is really hdf5-specific. Maybe I'd better write hdf5 build script..